ir/Cargo.toml

```diff
@@ -14,3 +14,4 @@ rust-version = "1.65"
 [dependencies]
 air-script-core = { package = "air-script-core", path="../air-script-core", version="0.2.0" }
 parser = { package = "air-parser", path = "../parser", version = "0.2.0" }
+miden-diagnostics = { git = "https://github.com/0xpolygonmiden/miden-diagnostics" }
```

Comment:
Hmm, I do think `miden-parsing-diagnostics` is too narrow, since the crate can be used on its own, and implies a relationship that is if anything the otherway around (i.e. `miden-parsing` depends on `miden-diagnostics`, but not vice-versa).

It's really intended for general compiler diagnostics, so the idea would be that any Miden project (in Rust) that includes a compilation component could lean on the crate for its diagnostics implementation. Not sure what name would work best, but ideally it wouldn't be super long, otherwise it becomes a bit unwieldy to use and refer to.

That said, unless those Miden rollup components are implemented in Rust, is a naming overlap going to be that problematic? We don't seem to have a naming convention that would distinguish between rollup components and non-rollup components yet in any case, but the same could probably be said for other Rust libraries we build. Might need to pin that down before we make a call here.

Reply:
Yeah - let's keep it as is for now. We can change this later if it becomes a problem (we did something similar with Miden VM crate which was originally just `miden` and now is `miden-vm`).